test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/MaxRequestBufferSizeTests.cs

```diff
@@ -165,7 +165,7 @@ public static IEnumerable<object[]> LargeUploadData
 
                     using (var reader = new StreamReader(stream, Encoding.ASCII))
                     {
-                        var response = reader.ReadToEnd();
+                        var response = await reader.ReadToEndAsync();
```

Comment:
Weird? Is it just because ReadToEnd() is slower, or is it actually getting blocked somehow?